pepper-apis/src/main/java/org/broadinstitute/ddp/model/familyhistory/FamilyHistoryRelation.java

```diff
@@ -0,0 +1,100 @@
+package org.broadinstitute.ddp.model.familyhistory;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.broadinstitute.ddp.model.user.UserProfile.SexType;
+
+/**
+ * Try to add some additional semantics to the labels used to refer to family relations
+ * We define sex and parents when the label suggests who they might be relative to another relation
+ */
+public enum FamilyHistoryRelation {
+    PATERNAL_GRANDFATHER(SexType.MALE, List.of()),
+    PATERNAL_GRANDMOTHER(SexType.FEMALE, List.of()),
+    MATERNAL_GRANDFATHER(SexType.MALE, List.of()),
+    MATERNAL_GRANDMOTHER(SexType.FEMALE, List.of()),
+    MOTHER(SexType.FEMALE,List.of(MATERNAL_GRANDFATHER,MATERNAL_GRANDMOTHER)),
+    FATHER(SexType.MALE,List.of(PATERNAL_GRANDFATHER, PATERNAL_GRANDMOTHER)),
+    PATERNAL_AUNT(SexType.FEMALE, List.of(), List.of(FATHER)),
```

Comment:
I'm guessing the `enum` here is for readability at a POC stage--would this sort of concept be something we manage relationally?

Reply:
For sure, this same relationships could be defined in the database. However, my sense overall, is that they are easier to work with as enums. Either way, if these definitions become unwieldy in code or would be desirable to allow end users to define their own, they can be easily migrated. Note that I tried to cover here all the relationships in Erin's mockups.